doc(~): updated doc (#156)
MyUnisoft/Redis: README.md

$ yarn add @myunisoft/redis
```

---

<p align="center">

>   There is multiple adapters in order to exploit the different abstractions available.

Reviewer: ```suggestion
  There are multiple adapters to exploit the different abstractions available.
```
